### ISSUE TITLE
 AttributeError: 'Distribution' object has no attribute 'convert_2to3_doctests'

### ISSUE THREAD

**Issue by ManPython** [ThibTrip/pangres#68]
```
\setuptools\command\build_py.py", line 132, in build_package_data
          srcfile in self.distribution.convert_2to3_doctests):
      AttributeError: 'Distribution' object has no attribute 'convert_2to3_doctests'
```
Can't install new version pangres
https://github.com/ThibTrip/pangres/releases/tag/v4.1.3
Befoere it works.

**Comment by ThibTrip** [comment]
Hi @ManPython, could you provide additional information? This could be related to the environment you are using.

I assume the following information could help reproduce the issue:

* Python version?
* Which Environment: Miniconda, Anaconda, venv, ...?
* OS
* Did you update an existing environment or did you create / recreate one?
* You could give me the output of `pip freeze` if that is OK with you

Perhaps you even have an environment file for me?

**Comment by ManPython** [comment]
Python 3.8.5 (tags/v3.8.5:580fbb0, Jul 20 2020, 15:57:54) [MSC v.1924 64 bit (AMD64)] on win32
Venv, 
W11
Stil this same env, previous version of pangers can install last not.

[log.txt](https://github.com/ThibTrip/pangres/files/10669999/log.txt)

Env is big.. and have many own packages. I can prepare new venv but this same error, soo.. I think no sense?

**Comment by ThibTrip** [comment]
@ManPython sorry for the delay. I cannot reproduce the problem but I think it is related to `setuptools` (I tried on Windows, Linux both with Python 3.8.5 and tried old versions of `setuptools` even ones before python 3.8.5 was released). I can see that the method `convert_2to3_doctests` is no longer in `...\site-packages\setuptools\command\build_py.py` since many versions.

Could you try with a newer version of setuptools?

**Comment by ManPython** [comment]
Setuptools 60.0.5 and this same.. wichid setuptools give You "pass", this same ver asmy? Strange.. 
Now have w11 after update from zero (format c), no changes around (pip cache is empty, etc.)
Using cached pangres-4.1.3.tar.gz (57 kB) - maybe try do new version with some changes.. can be on server problem?

**Comment by ThibTrip** [comment]
Hi @ManPython , I wanted to check in on your issue. I just uploaded a new version, you might want to try it again. TBH I don't see how uploading a new version would change anything but it's worth a try I guess.

In any case, I was thinking that you might want to try this:
* reinstalling `setuptools` with `pip install setuptools --force-reinstall`
* using a different environment manager. I am hopeful this could solve the problem. I personally had a problem with `virtualenv` (can't speak for `venv`) because of package names sort of "colliding together" (two libraries for samba which would install multiple packages). When I tried the same with `conda`, which I normally use, I had no problems. It's also what I use for testing `pangres` with many Python versions (from 3.6 all the way to 3.11). So yeah try your luck with another environment manager such as `conda`
